src/pages/index.js

```diff
@@ -56,6 +63,7 @@ export const FINALISABLE_PAGES = {
     finaliseText: 'finalise_supplier_invoice',
   },
   customerInvoice: {
+    checkForError: checkForCustomerInvoiceError,
```

Comment:
This + corresponding import means validation works on customer invoices finalisation again.

Reply:
As you've mentioned where the pruning was broken, the finalisation check was broken in https://github.com/sussol/mobile/pull/475/files